mne/decoding/tests/test_base.py

```diff
@@ -231,3 +231,20 @@ def test_cross_val_multiscore():
         clf.fit(X[train], y[train])
         scores_auc_manual.append(clf.score(X[test], y[test]))
     assert_array_equal(scores_auc, scores_auc_manual)
+
+    # indirectly test that cross_val_multiscore rightly detects the type of
+    # estimator and generates a StratifiedKFold for classiers and a KFold
+    # otherwise
+    X = np.random.randn(1000, 3)
+    y = np.r_[np.zeros(500), np.ones(500)]
+    clf = LogisticRegression(random_state=0)
+    reg = LinearRegression()
+    for cross_val in (cross_val_score, cross_val_multiscore):
+        manual = cross_val(clf, X, y, cv=StratifiedKFold(2))
+        auto = cross_val(clf, X, y, cv=2)
+        assert_array_equal(manual, auto)
+        assert_raises(ValueError, cross_val, clf, X, y, cv=KFold(2))
```

Comment:
are you preventing the use of KFold for classifiers? if so this sounds too much.